runtime/runtime/src/pipelining.rs

```diff
@@ -189,7 +189,8 @@ impl ReceiptPreparationPipeline {
                 | Action::Stake(_)
                 | Action::AddKey(_)
                 | Action::DeleteKey(_)
-                | Action::DeleteAccount(_) => {}
+                | Action::DeleteAccount(_)
+                | Action::DeployGlobalContract(_) => {}
```

Comment:
This needs to ensure that any of the `FunctionCall` actions involving the contract deployed by `DeployGlobalContract` in the current chunk are not pipelined (see the blocking logic for regular deploys.) Probably alright to punt on the implementation here to later, but putting it alongside the "no handling for these" chunk of actions runs a significant risk of forgetting about this.

Reply:
Global contract deploys only take effect in the next block, so those won't affect pipelining. The actual deploy (saving to the state) will be implemented with a separate receipt type later, and I will add proper handling for that in the pipelining code.